src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -67,13 +72,14 @@ private boolean isSameAccountNumbers(Organization organization) {
     Set<String> uniqueAccounts = organization.getAccounts().stream()
       .map(Account::getAccountNo)
       .collect(Collectors.toSet());
-
+    logger.debug("isSameAccountNumbers:: match unique accounts numbers size '{}' with accounts size '{}'", uniqueAccounts, organization.getAccounts().size());
```

Comment:
need to change uniqueAccounts -> uniqueAccounts .size()

Reply:
oh, thanks fixed